presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/EnvironmentExtender.java

```diff
@@ -13,9 +13,17 @@
  */
 package io.prestosql.tests.product.launcher.env.common;
 
+import com.google.common.collect.ImmutableList;
 import io.prestosql.tests.product.launcher.env.Environment;
 
+import java.util.List;
+
 public interface EnvironmentExtender
 {
     void extendEnvironment(Environment.Builder builder);
+
+    default List<EnvironmentExtender> getDependencies()
+    {
+        return ImmutableList.of();
```

Comment:
Can we document in which order extenders are applied to `Environment.Builder` ?

`dependencies, this` or `this, dependencies` ?

Reply:
as javadoc?